Project-Spaced-Repetition-Tracker/README.md

```diff
@@ -14,7 +14,6 @@ Note that when running locally, in order to open a web page which uses modules,
 
 We have also provided a `storage.js` file, which contains four functions to help with data storage. `storage.js` is a file containing four functions:
 
-- `getUserIds()`: when called, returns an array of strings, each of which is a user id
```

Comment:
I'm not sure about removing this - what are we expecting instead? For the trainees to hard-code their own 5 user IDs?

I suspect this may have the opposite effect of that described in the PR description - making people do _more_ elaborate things with users, rather than less?

Reply:
I think perhaps I didn't explain my thought properly: I agree, this will end up with trainees doing more elaborate things. But I think I'm fine with it as long as they have the 5 users from the req, and they don't fail simply because they've gone overboard with user generation (which seems unlikely to me?).

If we see the latter, then I'm happy to try the "precisely describe the requirements" path.